src/Microsoft.DotNet.Cli.Utils/CommandResolution/MSBuildProject.cs

```diff
@@ -135,7 +135,7 @@ private string GetLockFilePathFromProjectLockFileProperty()
                 .AllEvaluatedProperties
                 .Where(p => p.Name.Equals("ProjectAssetsFile"))
                 .Select(p => p.EvaluatedValue)
-                .FirstOrDefault(p => File.Exists(p));
+                .FirstOrDefault(p => Path.IsPathRooted(p) && File.Exists(p));
```

Comment:
Livar pointed this out to me. When that gets evaluated it returns multiple values. One of them is the relative path for the test project.